packages/react-button/src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -1,212 +1,19 @@
-        padding-top: var(--button-paddingTop);
-        position: relative;
-        text-decoration: none;
-        transition: var(--button-transition);
-        user-select: none;
-        vertical-align: middle;
-        white-space: var(--button-whiteSpace, nowrap);
-        width: var(--button-width);
-      }
-      .ms-Fabric--isFocusVisible &:focus::after {
-        border-bottom-left-radius: var(--button-borderBottomLeftRadius, var(--button-borderRadius));
-        border-bottom-right-radius: var(--button-borderBottomRightRadius, var(--button-borderRadius));
-        border-color: var(--button-focusColor, black);
-        border-style: solid;
-        border-top-left-radius: var(--button-borderTopLeftRadius, var(--button-borderRadius));
-        border-top-right-radius: var(--button-borderTopRightRadius, var(--button-borderRadius));
-        border-width: var(--button-focusWidth, 1.6px);
-        bottom: -1px;
-        box-shadow: 0 0 0 var(--button-focusInnerWidth, 1px) var(--button-focusInnerColor, white) inset;
-        content: "";
-        left: -1px;
-        position: absolute;
-        right: -1px;
-        top: -1px;
-      }
-      & > *:not(:first-child) {
-        margin-left: var(--button-contentGap);
-      }
-      &:hover {
-        background: var(--button-hovered-background, var(--button-background));
-        border-color: var(--button-hovered-borderColor, var(--button-borderColor));
-        box-shadow: var(--button-hovered-boxShadow, var(--button-boxShadow));
-        color: var(--button-hovered-contentColor, var(--button-contentColor));
-      }
-      &:hover .ms-Button-icon {
-        color: var(--button-hovered-iconColor, var(--button-iconColor));
-      }
-      &:active {
-        background: var(--button-pressed-background, var(--button-hovered-background));
-        border-color: var(--button-pressed-borderColor, var(--button-hovered-borderColor, var(--button-borderColor)));
-        box-shadow: var(--button-pressed-boxShadow, var(--button-hovered-boxShadow, var(--button-boxShadow)));
-        color: var(--button-pressed-contentColor, var(--button-hovered-contentColor, var(--button-contentColor)));
-        transform: var(--button-pressed-transform);
-        transition: var(--button-pressed-transition);
-      }
-      &:active .ms-Button-icon {
-        color: var(--button-pressed-iconColor, var(--button-iconColor));
-      }
-      &[aria-disabled=true] {
-        background-color: var(--button-disabled-background);
-        border-color: var(--button-disabled-borderColor);
-        box-shadow: var(--button-disabled-boxShadow);
-        color: var(--button-disabled-contentColor);
-        opacity: var(--button-disabled-opacity);
-        pointer-events: none;
-      }
-      &[aria-disabled=true] .ms-Button-icon {
-        color: var(--button-disabled-iconColor);
-      }
->
-  <span
-    className=
-
-        {
-          line-height: 1;
-        }
-  >
+<button>
+  <span>
```

Comment:
i'm confused why this change is here

Reply:
well that's confusing. I will try and figure this out.